src/main/java/redis/clients/jedis/Connection.java

```diff
@@ -271,13 +271,13 @@ public Long getIntegerReply() {
   }
 
   @SuppressWarnings("unchecked")
-  public List<Object> getRawObjectMultiBulkReply() {
```

Comment:
Keep this method, with `@Deprecated`, return `getUnflushedObjectMultiBulkReply()`.

Reply:
Done.